### PR TITLE
Set up the project with Webpack and Jest- group task

### LINKED ISSUE #18: [0.5pt] Set up the project with Webpack and Jest- group task
- Set up Gitflow
- Set up webpack
- Set up a JavaScript testing library (Jest)

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Team Partner

In this task, I need to do the following : 

- [x] Set up Gitflow
- [x] Set up webpack
- [x] Set up a JavaScript testing library (Jest)

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+function component() {
+  const element = document.createElement('div');
+
+  // Lodash, now imported by this script
+  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+  element.classList.add('hello');
+
+  return element;
+}
```
Comment (koomekevo) [inline src/index.js]: Refactor this into ES6 arrow function, that is 
` let component = () => {
       code goes here...
  }`

File: src/sum.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+  return a + b;
+}
```
Comment (koomekevo) [inline src/sum.js]: This one too can be refactored as ES6 arrow function, that is 
`let sum = (a, b) => a + b`

**Review by koomekevo — CHANGES_REQUESTED** [review]
### Changes required :recycle: 

Hi @benjp009,
You have followed the instructions to the latter, which required you to:
:heavy_check_mark: Set up Webpack 
:heavy_check_mark: Setup Jest.

However there are changes required as follows to move forward with the project.

File: src/index.js (hunk 1 above)
Comment (benjp009) [reply src/index.js]: Done

File: src/sum.js (hunk 2 above)
Comment (benjp009) [reply src/sum.js]: Done

**Comment by benjp009** [comment]
Hello @koomekevo, changes have been made to match ES6 :)

**Review by koomekevo — APPROVED** [review]
**Hi @benjp009 **

Your code has been finally approved. There is nothing else to say other than merge it :shipit: 

Happy coding :partying_face:

**Pull request merged**